docs/src/design/decoder/constraints.md

```diff
@@ -547,6 +547,12 @@ When `SPAN` or `RESPAN` operations is executed, one of the batch flags must be s
 (f_{span} + f_{respan}) - (f_{g1} + f_{g2} + f_{g4} + f_{g8}) = 0 \text{ | degree} = 5
 $$
 
+When neither `SPAN` nor `RESPAN` is executed, all batch flags must be set to $0$.
+
+> $$
+(1 - (f_{span} + f_{respan})) \cdot (1 - bc_0) \cdot (1 - bc_1) \cdot (1 - bc_2) = 0 \text{ | degree} = 8
+$$
```

Comment:
Great catch! But is this constraint correct? I think it would be satisfied as long as any of $bc_0$, $bc_1$, or $bc_2$ are equal to $1$ (rather than all of them) - no?

Reply:
Ah right! Should be fixed now